src/main/java/com/cloudbees/dockerpublish/DockerBuilder.java

```diff
@@ -301,7 +313,8 @@ private boolean buildAndTag() throws MacroEvaluationException, IOException, Inte
 				// we know the image name so apply the tags directly
 				while (lastResult.result && i.hasNext()) {
 					lastResult = executeCmd("docker tag --force=true " + image + " " + i.next());
-				}
+				}          
+                                processFingerprints(image);
```

Comment:
The file has both indentations inside.
If @carlossg and @michaelneale agree, I'll just replace tabs before the release. Otherwise tabs should be fixed